ui/src/containers/VolumeInformation.js

```diff
@@ -143,6 +167,8 @@ const VolumeInformation = props => {
         />
       ) : null}
 
+      {<LoaderContainer isLoading={isLoading} size="small" />}
```

Comment:
Need to find a better place for the Loader but fine for now

Reply:
ok! I did a lot of thinkings about where is the best place to put loader.  
Yeah, we will discuss later to find a good place.